packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.test.tsx

```diff
@@ -173,10 +173,14 @@ describe('CompleteResetPassword page', () => {
   });
 
   // TODO : check for metrics event when link is expired or damaged
```

Comment:
Might be a good time to do this TODO?

Reply:
We don't have metrics set up for the expired/damaged links yet. Looks like content-server logs error events, and I'll need to check on how we should handle that here. I'm not sure if we have something similar already set up for settings.

Reply:
Ah, do you mind filing an issue for that and throwing it into the next sprint? We can toss it out if we want to at sprint planning but it seems like we'd want that in place.

Reply:
Less than useful suggestion, but it'd be nice to start adding the Jira ticket numbers on each `TODO` probably so we know we have tickets filed for them and they aren't lingering unreported TODOs that won't get TODOne.

Looks like we currently have about 261 "TODO"s in the codebase and roughly 52 with "fxa-_nnnn_" Jira ticket numbers:

```sh
git grep "TODO: " packages | wc -l # 261
git grep -h "TODO: " packages | grep -iE "fxa-[[:digit:]]+" | wc -l # 52
```

Reply:
Nice little cleanup job there to see if the TODOs are still relevant, and if they actually got TODOne in the referenced ticket... 🧹 

Created ticket (https://mozilla-hub.atlassian.net/browse/FXA-7630) and updated comments.